File: test-output.py
import re
from playwright.sync_api import Playwright, sync_playwright, expect


def run(playwright: Playwright) -> None:
    browser = playwright.chromium.launch(headless=False)
    context = browser.new_context()
    page.get_by_role("link", name="RFC 2606").click()
    page.close()

    # ---------------------
    context.close()
    browser.close()


with sync_playwright() as playwright:
    run(playwright)

File: test-output.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch({
    headless: false
  });
  const context = await browser.newContext();
  await page.goto('https://www.browserscan.net/');
  await page.locator('span').filter({ hasText: 'Verizon Business' }).click();
  await page.goto('https://www.producthunt.com/');
  await page.close();

  // ---------------------
  await context.close();
  await browser.close();
})();